docker-bake.hcl

```diff
@@ -203,6 +203,7 @@ target "virtual-osbuild-ci" {
                         "nbd",
                         "nbd-cli",
                         "ostree",
+			"pacman",
```

Comment:
super nit: tabs vs spaces. We should add an `.editorconfig` file.

Reply:
Ok, spacified it :)